modules/ROOT/pages/design-import-files.adoc

```diff
@@ -1,31 +1,29 @@
-= Import Files into an API Project
+= Import Files into an API-Specification Project
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can import API specifications and RAML fragments into a project. There are three locations that you can import from: your computer, a location at a URL, and Anypoint Exchange.
+You can import API specifications and RAML fragments into an API-specification project.
```

Comment:
WHY do I want or need to?

Reply:
@kbostrom , users will know why they need to import files.